docs/release/trg-7/trg-7-01.md

```diff
@@ -75,22 +81,29 @@ Do the following changes:
 
 - Add both licenses to the "Declared Project Licenses" sections, see [example](https://github.com/eclipse-tractusx/sig-infra/blob/main/NOTICE.md)
 - Add the link to your repository
-- Add the link(s) to your SBOM, e.g. the DEPENDENCY file (one or more)
+- Add the link(s) to your SBOM, e.g. the DEPENDENCIES file (one or more)
 - Add information for third party content checks, if not covered by the Dash Tool (e.g. IP checks for icons, fonts, ...)
 
 [Further information](trg-7-04.md#checking-other-content-fonts-images-) and see the [Handbook#legaldoc-notice](https://www.eclipse.org/projects/handbook/#legaldoc-notice).
 
-### DEPENDENCY FILE
+### DEPENDENCIES FILE
 
 :::info
 
-Third-party dependencies need to be checked regularly to reflect your code changes. The DEPENDENCY file must be updated accordingly. This is recommended for every contribution (e.g. PR) whenever possible.
+Third-party dependencies need to be checked regularly to reflect your code changes. The DEPENDENCIES file must be updated
+accordingly. This is recommended for every contribution (e.g. PR).
 
 :::
 
 - Create it with the [Eclipse Dash License Tool](https://www.eclipse.org/projects/handbook/#ip-license-tool)
 
-If different technologies / package managers (e.g. npm and maven) are used you are free to have several dependency files. Use the naming convention DEPENDENCY_XYZ, e.g. DEPENDENCY_FRONTEND and DEPENDENCY_BACKEND.
+If different technologies / package managers (e.g. npm and maven) are used you are free to have several dependency files.
+Use the naming convention `DEPENDENCIES_XYZ`, e.g. `DEPENDENCIES_FRONTEND` and `DEPENDENCIES_BACKEND`.
+
+These files can be kept either checked in the git repository or published to a static location (e.g. GitHub Pages) and
+linked in the [NOTICE file](#notice-file) .
+It is advisable to run the check after every commit or during the nightly, surely it is mandatory to run it before the
+release, ensuring that no `rejected` or `restricted` dependencies are being part of delivered artifacts.
```

Comment:
```suggestion
These files can be kept either checked in the git repository or published to a static location (e.g. GitHub Pages) and linked in the [NOTICE file](#notice-file) during development against the main branch.
It is advisable to run the check after every commit or during the nightly, surely it is mandatory to run it before the release, ensuring that no `rejected` or `restricted` dependencies are being part of delivered artefacts.
```

Reply:
same

Reply:
```suggestion
These files can be kept either checked in the git repository or published to a static location (e.g. GitHub Pages) and
linked in the [NOTICE file](#notice-file).
It is advisable to run the check after every commit or during the nightly, surely it is mandatory to run it before the
release, ensuring that no `rejected` or `restricted` dependencies are being part of delivered artefacts.
```

Reply:
In the Java world (that's the base of most of the products here) "artifact" is used ([as maven states in their doc](https://maven.apache.org/repositories/artifacts.html)). I would prefer to keep it as it is

Reply:
sure, could please fix the other thing?

```suggestion
These files can be kept either checked in the git repository or published to a static location (e.g. GitHub Pages) and
linked in the [NOTICE file](#notice-file).
It is advisable to run the check after every commit or during the nightly, surely it is mandatory to run it before the
release, ensuring that no `rejected` or `restricted` dependencies are being part of delivered artifacts.
```

Reply:
sure, that little whitespace felt through the cracks, generally it would be easier to tackle single line comments unless they are related to a whole block.

done, btw :)